feat: complete refactor

isometry opened the pull request:
* Support commit of all staged or tracked files via `ghup content --staged` and `ghup content --all`.
* Remove redundant global flags and properly handle viper-mapping for flags used across multiple commands.
* Add `resolve` command to resolve commitish and optionally return matching branches/heads and tags.
* Fallback to retrieving the GitHub CLI auth token from `gh auth token`; disable with `--no-cli-token=false` or `GHUP_NO_CLI_TOKEN=0`.
* Fix automatic resolution of owner from origin to respect the current branch's remote rather than simply taking first github.com remote, which was subject to errors due to alpha-sorting.
* The `update-ref` command now takes a commitish as `--source`. The `--source-type` flag is now ignored and will be removed in a later update.
* The `tag` command now takes an optional `--commitish` flag used to determine the target for the tag; the `--branch` flag is ignored.
* Add basic acceptance tests.

Fixes: #73

pcanilho commented on cmd/content_test.go:
```diff
@@ -0,0 +1,369 @@
+		t.Fatalf("failed to write test file: %v", err)
+	}
+	defer func() {
+		_ = os.Remove(file1)
```
Given that this is a defer you can simplify and remove the whole block since `tmpDir` will be automatically deleted.
> TempDir returns a temporary directory for the test to use. The directory is automatically removed when the test and all its subtests complete.

isometry commented:
All tests passing:

```console
$ go test ./... -tags=acceptance -v
?   	github.com/nexthink-oss/ghup	[no test files]
=== RUN   TestAccContentCmd
=== RUN   TestAccContentCmd/Create_a_new_branch_with_content_updates
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Initial content test commit --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file1.txt:test-path/file1.txt --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file2.txt:test-path/file2.txt]
=== RUN   TestAccContentCmd/Update_existing_files_(idempotent_operation)
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Same content should be idempotent --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file1.txt:test-path/file1.txt --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file2.txt:test-path/file2.txt]
=== RUN   TestAccContentCmd/Update_a_file_with_new_content
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Adding a third file --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file3.txt:test-path/file3.txt]
=== RUN   TestAccContentCmd/Delete_a_file
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Deleting the second file --delete test-path/file2.txt]
=== RUN   TestAccContentCmd/Copy_a_file_from_the_same_branch
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Copying a file within the branch --copy test-content-sq8y130p:test-path/file1.txt:test-path/file1-copy.txt]
=== RUN   TestAccContentCmd/Copy_a_file_from_default_branch
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Copying a file from the default branch --copy main:README.md:test-path/readme-copy.md]
=== RUN   TestAccContentCmd/Update,_copy,_and_delete_in_one_operation
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Combined operation --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/delete_me.txt:test-path/new-file.txt --delete test-path/file3.txt --copy test-content-sq8y130p:test-path/file1.txt:test-path/another-copy.txt]
=== RUN   TestAccContentCmd/Create_a_PR_from_our_changes
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Update for PR creation --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file1.txt:test-path/update-for-pr.txt --pr-title Test PR from Content Command --pr-body This PR was created by the acceptance test]
=== RUN   TestAccContentCmd/Try_to_update_non-existent_branch_without_create_flag
    content_test.go:320: Running command with args: [-vvvv content --branch non-existent-branch-svya6b --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file1.txt:test-path/any-file.txt --create-branch=false]
=== RUN   TestAccContentCmd/Dry_run_mode_should_not_create_changes
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message This commit should not happen --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file1.txt:test-path/dry-run-file.txt --dry-run]
=== RUN   TestAccContentCmd/Force_update_even_if_file_content_is_the_same
    content_test.go:320: Running command with args: [-vvvv content --branch test-content-sq8y130p --message Force update with same content --update /var/folders/sz/y9t0qm6x3t7c8g8rx7_pdsf80000gp/T/TestAccContentCmd1138296580/001/file1.txt:test-path/file1.txt --force]
=== NAME  TestAccContentCmd
    root_test.go:35: cleanupResources for TestAccContentCmd
    root_test.go:49: Cleaned up branch "test-content-sq8y130p"
--- PASS: TestAccContentCmd (17.39s)
    --- PASS: TestAccContentCmd/Create_a_new_branch_with_content_updates (2.45s)
    --- PASS: TestAccContentCmd/Update_existing_files_(idempotent_operation) (1.20s)
    --- PASS: TestAccContentCmd/Update_a_file_with_new_content (1.33s)
    --- PASS: TestAccContentCmd/Delete_a_file (1.26s)
    --- PASS: TestAccContentCmd/Copy_a_file_from_the_same_branch (1.55s)
    --- PASS: TestAccContentCmd/Copy_a_file_from_default_branch (1.71s)
    --- PASS: TestAccContentCmd/Update,_copy,_and_delete_in_one_operation (2.13s)
    --- PASS: TestAccContentCmd/Create_a_PR_from_our_changes (2.67s)
    --- PASS: TestAccContentCmd/Try_to_update_non-existent_branch_without_create_flag (0.27s)
    --- PASS: TestAccContentCmd/Dry_run_mode_should_not_create_changes (0.61s)
    --- PASS: TestAccContentCmd/Force_update_even_if_file_content_is_the_same (1.29s)
=== RUN   TestAccResolveCmd
=== RUN   TestAccResolveCmd/Resolve_plain_commitish_to_SHA_only
    resolve_test.go:128: args: [-vvvv resolve main]
=== RUN   TestAccResolveCmd/Resolve_relative_commitish_to_SHA_only
    resolve_test.go:128: args: [-vvvv resolve main~1]
=== RUN   TestAccResolveCmd/Resolve_commitish_to_SHA_and_list_matching_branches
    resolve_test.go:128: args: [-vvvv resolve main --branches]
=== RUN   TestAccResolveCmd/Resolve_commitish_to_SHA_and_list_matching_tags
    resolve_test.go:128: args: [-vvvv resolve main --tags]
=== RUN   TestAccResolveCmd/Resolve_non-existing_commitish
    resolve_test.go:128: args: [-vvvv resolve thisdefinitelydoesnotexist]
=== NAME  TestAccResolveCmd
    root_test.go:35: cleanupResources for TestAccResolveCmd
    root_test.go:41: Cleaned up tag "test-tag-vn1pwe0d"
    root_test.go:49: Cleaned up branch "test-content-1ta3oc3a"
--- PASS: TestAccResolveCmd (3.88s)
    --- PASS: TestAccResolveCmd/Resolve_plain_commitish_to_SHA_only (0.29s)
    --- PASS: TestAccResolveCmd/Resolve_relative_commitish_to_SHA_only (0.30s)
    --- PASS: TestAccResolveCmd/Resolve_commitish_to_SHA_and_list_matching_branches (0.52s)
    --- PASS: TestAccResolveCmd/Resolve_commitish_to_SHA_and_list_matching_tags (0.46s)
    --- PASS: TestAccResolveCmd/Resolve_non-existing_commitish (0.25s)
=== RUN   TestAccTagCmd
=== RUN   TestAccTagCmd/no_args
    tag_test.go:155: args: [-vvvv tag]
=== RUN   TestAccTagCmd/create_annotated_tag
    tag_test.go:155: args: [-vvvv tag tag-b1dhfp]
=== RUN   TestAccTagCmd/create_annotated_tag_is_idempotent
    tag_test.go:155: args: [-vvvv tag tag-b1dhfp]
=== RUN   TestAccTagCmd/create_lightweight_tag_with_clashing_annotated_tag
    tag_test.go:155: args: [-vvvv tag tag-b1dhfp --lightweight]
=== RUN   TestAccTagCmd/force_replace_annotated_tag_with_lightweight
    tag_test.go:155: args: [-vvvv tag tag-b1dhfp --lightweight --force]
=== RUN   TestAccTagCmd/create_lightweight_tag
    tag_test.go:155: args: [-vvvv tag light-tag-b1dhfp --lightweight]
=== RUN   TestAccTagCmd/create_lightweight_tag_is_idempotent
    tag_test.go:155: args: [-vvvv tag light-tag-b1dhfp --lightweight]
=== RUN   TestAccTagCmd/create_annotated_tag_with_clashing_lightweight_tag
    tag_test.go:155: args: [-vvvv tag light-tag-b1dhfp]
=== RUN   TestAccTagCmd/force_replace_lightweight_tag_with_annotated
    tag_test.go:155: args: [-vvvv tag light-tag-b1dhfp --force]
=== NAME  TestAccTagCmd
    root_test.go:35: cleanupResources for TestAccTagCmd
    root_test.go:41: Cleaned up tag "tag-b1dhfp"
    root_test.go:41: Cleaned up tag "light-tag-b1dhfp"
--- PASS: TestAccTagCmd (10.75s)
    --- PASS: TestAccTagCmd/no_args (0.16s)
    --- PASS: TestAccTagCmd/create_annotated_tag (1.54s)
    --- PASS: TestAccTagCmd/create_annotated_tag_is_idempotent (1.10s)
    --- PASS: TestAccTagCmd/create_lightweight_tag_with_clashing_annotated_tag (1.04s)
    --- PASS: TestAccTagCmd/force_replace_annotated_tag_with_lightweight (1.34s)
    --- PASS: TestAccTagCmd/create_lightweight_tag (1.50s)
    --- PASS: TestAccTagCmd/create_lightweight_tag_is_idempotent (0.95s)
    --- PASS: TestAccTagCmd/create_annotated_tag_with_clashing_lightweight_tag (0.90s)
    --- PASS: TestAccTagCmd/force_replace_lightweight_tag_with_annotated (1.65s)
=== RUN   TestAccUpdateRefCmd
    updateref_test.go:76: Creating initial test refs
    updateref_test.go:94: Created test ref refs/heads/test-update-ref-branch-nzjpqg75 pointing to ee18427a4079cccdeae849168a1115b02dc8cf4a
    updateref_test.go:94: Created test ref refs/tags/test-update-ref-tag-if3gsc8x pointing to ee18427a4079cccdeae849168a1115b02dc8cf4a
=== RUN   TestAccUpdateRefCmd/Idempotent_branch_update
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source main refs/heads/test-update-ref-branch-nzjpqg75]
=== RUN   TestAccUpdateRefCmd/Idempotent_tag_update
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source main refs/tags/test-update-ref-tag-if3gsc8x]
=== RUN   TestAccUpdateRefCmd/New_branch_pointing_to_default_branch
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source main refs/heads/test-update-ref-branch-rhu6xtj6]
=== RUN   TestAccUpdateRefCmd/New_tag_pointing_to_default_branch
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source main refs/tags/test-update-ref-tag-guyt37c5]
=== RUN   TestAccUpdateRefCmd/Update_multiple_refs_at_once
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source main refs/tags/test-update-ref-tag-if3gsc8x refs/heads/test-update-ref-branch-nzjpqg75]
=== RUN   TestAccUpdateRefCmd/Update_using_a_commit_SHA_as_source
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source ee18427a4079cccdeae849168a1115b02dc8cf4a --force refs/tags/test-update-ref-tag-if3gsc8x]
=== RUN   TestAccUpdateRefCmd/Error_-_missing_source
    updateref_test.go:218: Running command with args: [-vvvv update-ref refs/tags/test-update-ref-tag-if3gsc8x]
=== RUN   TestAccUpdateRefCmd/Error_-_missing_targets
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source main]
=== RUN   TestAccUpdateRefCmd/Error_-_non-existent_source
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source refs/heads/this-branch-does-not-exist-w9th0z4k refs/tags/test-update-ref-tag-if3gsc8x]
=== RUN   TestAccUpdateRefCmd/Source_using_different_format_(unqualified)
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source main --source-type heads --force refs/tags/test-update-ref-tag-if3gsc8x]
=== RUN   TestAccUpdateRefCmd/Target_using_different_format_(unqualified)
    updateref_test.go:218: Running command with args: [-vvvv update-ref --source main --target-type tags --force test-update-ref-tag-if3gsc8x]
=== NAME  TestAccUpdateRefCmd
    root_test.go:35: cleanupResources for TestAccUpdateRefCmd
    root_test.go:41: Cleaned up tag "test-update-ref-tag-if3gsc8x"
    root_test.go:41: Cleaned up tag "test-update-ref-tag-guyt37c5"
    root_test.go:49: Cleaned up branch "test-update-ref-branch-nzjpqg75"
    root_test.go:49: Cleaned up branch "test-update-ref-branch-rhu6xtj6"
--- PASS: TestAccUpdateRefCmd (9.74s)
    --- PASS: TestAccUpdateRefCmd/Idempotent_branch_update (0.83s)
    --- PASS: TestAccUpdateRefCmd/Idempotent_tag_update (0.90s)
    --- PASS: TestAccUpdateRefCmd/New_branch_pointing_to_default_branch (0.82s)
    --- PASS: TestAccUpdateRefCmd/New_tag_pointing_to_default_branch (0.84s)
    --- PASS: TestAccUpdateRefCmd/Update_multiple_refs_at_once (1.28s)
    --- PASS: TestAccUpdateRefCmd/Update_using_a_commit_SHA_as_source (0.81s)
    --- PASS: TestAccUpdateRefCmd/Error_-_missing_source (0.00s)
    --- PASS: TestAccUpdateRefCmd/Error_-_missing_targets (0.28s)
    --- PASS: TestAccUpdateRefCmd/Error_-_non-existent_source (0.30s)
    --- PASS: TestAccUpdateRefCmd/Source_using_different_format_(unqualified) (0.78s)
    --- PASS: TestAccUpdateRefCmd/Target_using_different_format_(unqualified) (0.77s)
PASS
ok  	github.com/nexthink-oss/ghup/cmd	42.375s
=== RUN   TestSplitUpdateSpec
=== RUN   TestSplitUpdateSpec/Single_file
=== RUN   TestSplitUpdateSpec/Source_and_target
=== RUN   TestSplitUpdateSpec/Empty_parameter
=== RUN   TestSplitUpdateSpec/Missing_source
=== RUN   TestSplitUpdateSpec/Missing_target
=== RUN   TestSplitUpdateSpec/Alternate_separator
--- PASS: TestSplitUpdateSpec (0.00s)
    --- PASS: TestSplitUpdateSpec/Single_file (0.00s)
    --- PASS: TestSplitUpdateSpec/Source_and_target (0.00s)
    --- PASS: TestSplitUpdateSpec/Empty_parameter (0.00s)
    --- PASS: TestSplitUpdateSpec/Missing_source (0.00s)
    --- PASS: TestSplitUpdateSpec/Missing_target (0.00s)
    --- PASS: TestSplitUpdateSpec/Alternate_separator (0.00s)
PASS
ok  	github.com/nexthink-oss/ghup/internal/local	0.402s
=== RUN   TestCommitMessage
=== RUN   TestCommitMessage/Empty_message
=== RUN   TestCommitMessage/Single_line_message
=== RUN   TestCommitMessage/Multi-line_message
--- PASS: TestCommitMessage (0.00s)
    --- PASS: TestCommitMessage/Empty_message (0.00s)
    --- PASS: TestCommitMessage/Single_line_message (0.00s)
    --- PASS: TestCommitMessage/Multi-line_message (0.00s)
PASS
ok  	github.com/nexthink-oss/ghup/internal/remote	(cached)
=== RUN   TestGithubActionsBranch
=== RUN   TestGithubActionsBranch/PR_context
=== RUN   TestGithubActionsBranch/Other_context
=== RUN   TestGithubActionsBranch/Not_a_branch
--- PASS: TestGithubActionsBranch (0.00s)
    --- PASS: TestGithubActionsBranch/PR_context (0.00s)
    --- PASS: TestGithubActionsBranch/Other_context (0.00s)
    --- PASS: TestGithubActionsBranch/Not_a_branch (0.00s)
=== RUN   TestGithubActionsContext
=== RUN   TestGithubActionsContext/Valid_context
=== RUN   TestGithubActionsContext/PR_context
=== RUN   TestGithubActionsContext/Tag_context
=== RUN   TestGithubActionsContext/No_repository
--- PASS: TestGithubActionsContext (0.00s)
    --- PASS: TestGithubActionsContext/Valid_context (0.00s)
    --- PASS: TestGithubActionsContext/PR_context (0.00s)
    --- PASS: TestGithubActionsContext/Tag_context (0.00s)
    --- PASS: TestGithubActionsContext/No_repository (0.00s)
=== RUN   TestIsCommitHash
=== RUN   TestIsCommitHash/Valid_short_hash
=== RUN   TestIsCommitHash/Valid_long_hash
=== RUN   TestIsCommitHash/Invalid_hash_with_non-hex_characters
=== RUN   TestIsCommitHash/Invalid_hash_too_short
=== RUN   TestIsCommitHash/Invalid_hash_too_long
=== RUN   TestIsCommitHash/Empty_string
--- PASS: TestIsCommitHash (0.00s)
    --- PASS: TestIsCommitHash/Valid_short_hash (0.00s)
    --- PASS: TestIsCommitHash/Valid_long_hash (0.00s)
    --- PASS: TestIsCommitHash/Invalid_hash_with_non-hex_characters (0.00s)
    --- PASS: TestIsCommitHash/Invalid_hash_too_short (0.00s)
    --- PASS: TestIsCommitHash/Invalid_hash_too_long (0.00s)
    --- PASS: TestIsCommitHash/Empty_string (0.00s)
=== RUN   TestQualifiedRefName
=== RUN   TestQualifiedRefName/Fully-qualified_ref
=== RUN   TestQualifiedRefName/Fully-qualified_ref#01
=== RUN   TestQualifiedRefName/Partially-qualified_ref
=== RUN   TestQualifiedRefName/Partially-qualified_ref#01
=== RUN   TestQualifiedRefName/Unqualified_ref_with_heads_default_#1
=== RUN   TestQualifiedRefName/Unqualified_ref_with_heads_default_#2
=== RUN   TestQualifiedRefName/Unqualified_ref_with_tags_default
=== RUN   TestQualifiedRefName/Partially-qualified_ref_with_mismatched_default_prefix
=== RUN   TestQualifiedRefName/Invalid_ref_name_with_control_characters
=== RUN   TestQualifiedRefName/Invalid_ref_name_with_consecutive_dots
=== RUN   TestQualifiedRefName/Invalid_ref_name_with_trailing_dot
=== RUN   TestQualifiedRefName/Invalid_ref_name_with_leading_slash
=== RUN   TestQualifiedRefName/Invalid_ref_name_with_trailing_slash
=== RUN   TestQualifiedRefName/Invalid_ref_name_with_sequence_@{
--- PASS: TestQualifiedRefName (0.00s)
    --- PASS: TestQualifiedRefName/Fully-qualified_ref (0.00s)
    --- PASS: TestQualifiedRefName/Fully-qualified_ref#01 (0.00s)
    --- PASS: TestQualifiedRefName/Partially-qualified_ref (0.00s)
    --- PASS: TestQualifiedRefName/Partially-qualified_ref#01 (0.00s)
    --- PASS: TestQualifiedRefName/Unqualified_ref_with_heads_default_#1 (0.00s)
    --- PASS: TestQualifiedRefName/Unqualified_ref_with_heads_default_#2 (0.00s)
    --- PASS: TestQualifiedRefName/Unqualified_ref_with_tags_default (0.00s)
    --- PASS: TestQualifiedRefName/Partially-qualified_ref_with_mismatched_default_prefix (0.00s)
    --- PASS: TestQualifiedRefName/Invalid_ref_name_with_control_characters (0.00s)
    --- PASS: TestQualifiedRefName/Invalid_ref_name_with_consecutive_dots (0.00s)
    --- PASS: TestQualifiedRefName/Invalid_ref_name_with_trailing_dot (0.00s)
    --- PASS: TestQualifiedRefName/Invalid_ref_name_with_leading_slash (0.00s)
    --- PASS: TestQualifiedRefName/Invalid_ref_name_with_trailing_slash (0.00s)
    --- PASS: TestQualifiedRefName/Invalid_ref_name_with_sequence_@{ (0.00s)
=== RUN   TestBuildCommitMessage
=== RUN   TestBuildCommitMessage/No_message_and_no_trailers
=== RUN   TestBuildCommitMessage/Only_message
=== RUN   TestBuildCommitMessage/Message_with_long_title
=== RUN   TestBuildCommitMessage/Message_with_trailers
=== RUN   TestBuildCommitMessage/Message_with_author_trailer_disabled
=== RUN   TestBuildCommitMessage/Only_trailers
=== RUN   TestBuildCommitMessage/Multiple_trailers
--- PASS: TestBuildCommitMessage (0.00s)
    --- PASS: TestBuildCommitMessage/No_message_and_no_trailers (0.00s)
    --- PASS: TestBuildCommitMessage/Only_message (0.00s)
    --- PASS: TestBuildCommitMessage/Message_with_long_title (0.00s)
    --- PASS: TestBuildCommitMessage/Message_with_trailers (0.00s)
    --- PASS: TestBuildCommitMessage/Message_with_author_trailer_disabled (0.00s)
    --- PASS: TestBuildCommitMessage/Only_trailers (0.00s)
    --- PASS: TestBuildCommitMessage/Multiple_trailers (0.00s)
=== RUN   TestBuildTrailers
=== RUN   TestBuildTrailers/No_trailers
=== RUN   TestBuildTrailers/Author_trailer_only
=== RUN   TestBuildTrailers/Author_trailer_disabled
=== RUN   TestBuildTrailers/Multiple_trailers
=== RUN   TestBuildTrailers/Only_additional_trailers
--- PASS: TestBuildTrailers (0.00s)
    --- PASS: TestBuildTrailers/No_trailers (0.00s)
    --- PASS: TestBuildTrailers/Author_trailer_only (0.00s)
    --- PASS: TestBuildTrailers/Author_trailer_disabled (0.00s)
    --- PASS: TestBuildTrailers/Multiple_trailers (0.00s)
    --- PASS: TestBuildTrailers/Only_additional_trailers (0.00s)
PASS
ok  	github.com/nexthink-oss/ghup/internal/util	(cached)
=== RUN   TestChoiceFlag_Set
=== RUN   TestChoiceFlag_Set/Valid_choice
=== RUN   TestChoiceFlag_Set/Invalid_choice
--- PASS: TestChoiceFlag_Set (0.00s)
    --- PASS: TestChoiceFlag_Set/Valid_choice (0.00s)
    --- PASS: TestChoiceFlag_Set/Invalid_choice (0.00s)
=== RUN   TestChoiceFlag_String
--- PASS: TestChoiceFlag_String (0.00s)
=== RUN   TestChoiceFlag_Type
--- PASS: TestChoiceFlag_Type (0.00s)
=== RUN   TestNewChoiceFlag
--- PASS: TestNewChoiceFlag (0.00s)
PASS
ok  	github.com/nexthink-oss/ghup/pkg/choiceflag	(cached)
```

isometry commented on cmd/root.go:
```diff
@@ -50,108 +41,30 @@ func Execute() {
+	persistentFlags.StringP("repo", "r", localRepo.Name, "repository `name`")
+	persistentFlags.Bool("no-cli-token", false, "disable fallback to GitHub CLI Token")
+	persistentFlags.CountP("verbose", "v", "increase verbosity``")
 
```
This is an intentional, and dumb, hack to have cobra *not* print `--verbose count`, but rather just `--verbose` in its help output :-)
